src/DOMelements.js

```diff
@@ -0,0 +1,84 @@
+import { fetchWeatherAsync, fetchFlickrPhoto, toggleLoader } from "./index.js";
+import { getCountryCode } from "./countryCode.js";
+// search button and array of divs with data
+let button = document.querySelector("#citySearchButton");
+export let dataDivs = [...document.querySelectorAll(".data")];
+
+// adds search button event handler
+button.addEventListener("click", () => {
+  toggleLoader();
+  let searchQuery = document.querySelector("#cityNameInput").value;
+  let infoArray = searchQuery.split(",");
+  let nameOfCity = infoArray[0];
+  // city name without the space in front of the name
+  let nameOfCountry = infoArray[infoArray.length - 1].replace(" ", "");
+  let countryCode = getCountryCode(nameOfCountry);
+  let query = `${nameOfCity},${countryCode}`;
+
+  let units = getUnit();
+
+  //clear search
+  document.querySelector("#cityNameInput").value = "";
+  // fetches the weather data
+  fetchWeatherAsync(query, units);
+  // fetches the city photo
+  fetchFlickrPhoto(nameOfCity);
+  menuToggle("menu-overlay");
+});
+
+// celcius
+let metricButton = document.querySelector(".metric");
+// fahrenheit
+let imperialButton = document.querySelector(".imperial");
+let unitsButtons = [metricButton, imperialButton];
+
+function getUnit() {
```

Comment:
Use es6 in place of es5 `function`.
```suggestion
const getUnit = () => {
```